bundles/org.openhab.binding.homewizard/src/main/resources/OH-INF/i18n/homewizard.properties

```diff
@@ -5,25 +5,48 @@ addon.homewizard.description = This binding provides access to the data provided
 
 # thing types
 
+thing-type.homewizard.energy_socket.label = HomeWizard Energysocket
+thing-type.homewizard.energy_socket.description = This thing provides the measurement data that is available through the http interface of a HomeWizard Energysocket.
```

Comment:
This seems duplicated from the next two lines?

Reply:
Result of duplice thing declaration, fixed.